src/test/java/tr/com/srdc/cda2fhir/testutil/ClinicalDocumentMetadataGenerator.java

```diff
@@ -0,0 +1,110 @@
+package tr.com.srdc.cda2fhir.testutil;
+
+import org.hl7.fhir.utilities.xhtml.HierarchicalTableGenerator.Title;
+import org.openhealthtools.mdht.uml.cda.ClinicalDocument;
+import org.openhealthtools.mdht.uml.hl7.datatypes.CE;
+import org.openhealthtools.mdht.uml.hl7.datatypes.CS;
+import org.openhealthtools.mdht.uml.hl7.datatypes.II;
+import org.openhealthtools.mdht.uml.hl7.datatypes.INT;
+import org.openhealthtools.mdht.uml.hl7.datatypes.ST;
+import org.openhealthtools.mdht.uml.hl7.datatypes.TS;
+
+public class ClinicalDocumentMetadataGenerator {
+	ClinicalDocument doc;
+	static final public String DEFAULT_REALM_CODE = "US";
+	static final public String DEFAULT_ID_ROOT = "1.2.840.114350.1.13.88.3.7.8.688883.41197285";
+	static final public String DEFAULT_ASSN_AUTH = "EPC";
+	static final public String DEFAULT_CODE_CODE = "34133-9";
+	static final public String DEFAULT_CODE_SYSTEM = "2.16.840.1.113883.6.1";
+	static final public String DEFAULT_CODE_SYSTEM_NAME= "LOINC";
+	static final public String DEFAULT_CODE_DISPLAY = "Summarization of Episode Note";
+	static final public String DEFAULT_TITLE = "Patient Health Summary";
+	static final public String DEFAULT_EFFCT_DTTM = "20190214161413-0500";
+	static final public String DEFAULT_CONF_CODE_CODE = "N";
+	static final public String DEFAULT_CONF_CODE_SYSTEM = "2.16.840.1.113883.5.25";
+	static final public String DEFAULT_CONF_CODE_DSP = "Normal";
+	static final public String DEFAULT_LANG_CODE = "en-US";
+	static final public String DEFAULT_SET_ID_EXT = "7a8b50d2-309d-11e9-a581-5102195700f0";
```

Comment:
probably should have a default id of a generated uuid.

Reply:
ok... what element would that go in inside the cda header? templateId?

Reply:
in the set ID? isn't "7a8b50d2-309d-11e9-a581-5102195700f0" a generated uuid?

Reply:
it is, I was just saying make it "########-####-####..." or something like that.